receiver/statsdreceiver/metadata.yaml

```diff
@@ -0,0 +1,7 @@
+type: statsdreceiver
+
+status:
+  class: receiver
+  stability:
+    beta: [metrics]
+  distributions: [contrib, splunk]
```

Comment:
Needs another rebase, `sumo` was added. Sorry for all the churn @my-git9

Reply:
I fixed the conflict with the github UI, lemme know if that works.

Reply:
This is still unresolved. sumo is added in the doc only, not here

Reply:
I pushed 2 more commits to get this synchronized